api/src/main/java/io/minio/MinioClient.java

```diff
@@ -8156,19 +8244,29 @@ protected String uploadPart(
    * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
    * @throws XmlParserException thrown to indicate XML parsing error.
    */
-  protected String uploadPartCopy(
+  private String uploadPartCopy(
```

Comment:
1. We are providing core level support by `protected`. Any reason ti change it to `private`? 
2. Create one more method with `Multimap` than change it here.